packages/amplify_auth_cognito/example/ios/unit_tests/amplify_auth_cognito_tests.swift

```diff
@@ -281,14 +282,34 @@ class amplify_auth_cognito_tests: XCTestCase {
         XCTAssertNoThrow(try FlutterSignUpRequest.validate(dict: rawData))
     }
     
-    func test_signUpFormatAttributes() {
+    func test_signUpFormatAttributesWithUserAttributes() {
```

Comment:
*nit:* The name on each of these could probably be updated to remove the `FormatAttributes`
```suggestion
    func test_signUpWithUserAttributes() {
```